oak_functions_service/benches/wasm_benchmark.rs

```diff
@@ -0,0 +1,128 @@
+//
+// Copyright 2024 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use std::sync::Arc;
+
+use bytes::Bytes;
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion, Throughput};
+use hashbrown::HashMap;
+use oak_functions_abi::Request;
+use oak_functions_service::{
+    logger::StandaloneLogger,
+    lookup::{Data, LookupDataManager},
+    wasm::WasmHandler,
+};
+use rand::{rngs::SmallRng, Rng, SeedableRng};
+
+fn bench_invoke_echo(c: &mut Criterion) {
+    let test_state = create_test_state_with("echo");
```

Comment:
For easier readability, consider naming this `create_test_state_with_name` or adding explicitly `create_test_state_with(/* name= */ "echo")`. Without that, when I first read it I thought the argument was more like a test payload or test data not a test name.

Reply:
Done.